project_task_description_template/tests/test_project_task.py

```diff
@@ -0,0 +1,91 @@
+from odoo.tests import Form
+from odoo.tests.common import TransactionCase
```

Comment:
Please use `BaseCommon`
`from odoo.addons.base.tests.common import BaseCommon`.

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0#tasks-to-do-in-the-migration